CMakeLists.txt

```diff
@@ -34,7 +34,15 @@ option(WERROR "Treat compiler warnings as errors" OFF)
 
 set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake/Modules")
 
-option(BUILD_IDLC "Build IDL preprocessor" ON)
+if (CMAKE_CROSSCOMPILING)
+  option(BUILD_IDLC "Build IDL preprocessor (default:on, default-crosscompiling:off)" OFF)
+  option(BUILD_DDSCONF "Build DDSCONF buildtool (default:on, default-crosscompiling:off)" OFF)
+  option(INSTALL_DDSCONF "Install DDSCONF buildtool (default:off, default-crosscompiling:off)" OFF)
```

Comment:
I think we can use the same mechanism as we do for `idlc`? Maybe I'm overlooking a detail? I'd prefer the `INSTALL_DDSCONF` option is left out unless absolutely necessary.